action.yml

```diff
@@ -292,6 +292,6 @@ runs:
             repo: context.repo.repo,
             body: 'Testing Farm [request](${{ inputs.api_url }}/requests/${{ steps.sched_test.outputs.req_id }})' +
                   ' for ${{ inputs.compose }}/${{ inputs.copr_artifacts }} regression testing has been created.' +
-                  '\nOnce finished, results should be available [here](${{ steps.sched_test.outputs.results_url }}/).' +
-                  '\n[Full pipeline log](${{ steps.sched_test.outputs.results_url }}/pipeline.log).'
+                  '\nOnce finished, results should be available [here](${{ steps.final_state.outputs.results_url }}/).' +
```

Comment:
This will break results url publishing for all pipelines that don't update pull request status (like our pipelines).
Please make this conditional on `inputs.update_pull_request_status` and keep old logic for those pipelines who never run final_state step.

Reply:
What about making `final_state` general? I don't see any reason, why it should be available only for `inputs.update_pull_request_status`. It provides everything.

I don't follow, why we have marked `final_state` available only for updating PR status.

Reply:
At least `tf_url` is the correct one for the public ranch as for RedHat ranch.